### ISSUE TITLE
Docker container benötigt application.yml für Build

### ISSUE THREAD

**Issue by penguineer** [netz39/netz39fees#25]
Der Docker-Container lässt sich nur mit der endgültigen config/application.yml bauen (sonst Fehler im letzten Schritt). 

Das heißt in der Konsequenz, dass wir nur vorkonfigurierte Container ablegen können und eine Docker Registry (um Build-Umgebungen im Deployment-System zu vermeiden) nicht nutzbar sind.

Es sollte möglich sein, die application.yml in den Container zu mounten. Dazu muss wahrscheinlich das letzte RUN so umgebaut werden, dass es Teil des EXEC ist und beim Start des Containers ausgeführt wird.

**Comment by penguineer** [comment]
Alternativ könnte man da ein Dummy-File ablegen und für die Runtime per mount überschreiben?

**Comment by okirmis** [comment]
> Alternativ könnte man da ein Dummy-File ablegen und für die Runtime per mount überschreiben?

Das ginge, der "container-way" wäre vmtl., in der `application.yml` auf Umgebungsvariablen zuzugreifen und den Container darüber zu konfigurieren.

**Comment by penguineer** [comment]
Ja, sehr gern auch so, solange die Konfiguration nicht Teil des Images sein muss.

**Comment by okirmis** [comment]
Ich bau da mal einen PR für.

**Comment by okirmis** [comment]
Bitte mal drüber schauen: #26

**Comment by penguineer** [comment]
Nach dem Test von #26 fällt auf, dass das nicht das eigentliche Problem ist. Der Container sollte sich ohne eine fertige Konfiguration bauen lassen, das heißt der letzte Schritt muss in die Ausführung des Containers verschoben werden.
